task-sdk/pyproject.toml

```diff
@@ -46,6 +46,7 @@ classifiers = [
 ]
 
 dependencies = [
+    "apache-airflow-core<=3.1.0,>=3.0.0",
```

Comment:
@kaxil @ashb -> do you also feel this one should be as well?  I guess the "not plnning breaking changes" applies here as well - just wanted to make sure :).

Reply:
Yup, this one is good too.

FInish this split is the main thing Kaxil and I will be working on for 3.1/Task-SDK 1.1